B/BPNET/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "BPNET"
+version = v"0.0.2"
+
+# Collection of sources required to complete build
+sources = [
+GitSource("https://github.com/cometscome/BPNET.git", "207fd1d739b8cadcf243fc1bd7e1e8d7d28af392"),
+    DirectorySource("./bundled")
+]
+
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd BPNET/ 
+cmake -B build -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+cmake --build build --parallel ${nproc}
+cmake --install build
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+platforms = expand_gfortran_versions(platforms)
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("trainbin2ASCII.x", :bpnet_trainbin2ASCII),
```

Comment:
Do you really need the bpnet prefix for all the variables? They'd be defined in the bpnet_jll package, there'd be lots of bpnet around.

Reply:
No I do not need the prefix. I removed it. Thank you.